keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -49,9 +51,11 @@ class DefaultConfiguration {
   fun idGenerator(): ULID = ULID()
 
   @Bean
+  @Primary
   fun objectMapper(): ObjectMapper = configuredObjectMapper()
 
   @Bean
+  @Qualifier("yamlMapper")
```

Comment:
To differentiate from the (JSON) `ObjectMapper` above since both beans qualify as `YAMLMapper` is a sub-class. This eliminates any ambiguity.